modules/server/main.tf

```
@@ -58,11 +58,9 @@ resource "azurerm_mssql_firewall_rule" "this" {
 }
 
 resource "azurerm_mssql_server_extended_auditing_policy" "this" {
-  server_id                               = azurerm_mssql_server.this.id
-  storage_endpoint                        = azurerm_storage_account.this.primary_blob_endpoint
-  storage_account_access_key              = azurerm_storage_account.this.primary_access_key
-  storage_account_access_key_is_secondary = false
-  retention_in_days                       = 7
+  server_id              = azurerm_mssql_server.this.id
+  log_monitoring_enabled = true
```

```suggestion
  log_monitoring_enabled = true
  storage_endpoint                        = azurerm_storage_account.this.primary_blob_endpoint
  storage_account_access_key              = azurerm_storage_account.this.primary_access_key
  storage_account_access_key_is_secondary = false
```

@hknutsen @kov117 What are your thoughs on enabling log monitoring in addition to sending the extended server auditing policy to storage account? Or is that concidered as an unneccesary duplication?

---

Unnecessary duplication imo. 🙂

---

Resolving this conversation without suggested changes.